ui/src/components/forms/Combobox.tsx

```diff
@@ -51,7 +54,9 @@ export default function Combobox<T extends IFilterable>(
       <div className="relative flex w-full flex-row">
         <C.Input
           id={id}
-          className="text-gray-900 bg-gray-50 border-gray-300 w-full rounded-md border py-2 pl-3 pr-10 shadow-sm focus:border-violet-500 focus:outline-none focus:ring-1 focus:ring-violet-500 sm:text-sm"
+          className={twMerge(
```

Comment:
does `twMerge` replace the `classNames` util helper?

Reply:
@markphelps I think so? Was the `classNames` util helper for handling overrides of tailwind styles as well? If so, it didn't seem to be working for me for some reason 🤔. This [article](https://dev.to/diogoko/overriding-tailwind-classes-in-react-4po3) explains why `twMerge` should be used.

Reply:
i think `twMerge` is like the functionality of `classNames` + handles tailwind specifically. `classNames` is just a way to dynamically apply classes given a boolean condition. so it might make sense for us to favor `twMerge` instead of `classNames` in the future as it has the added ability of working with tw classes like we might expect (last one wins).

So maybe the rule should be, if we find ourselves needing to use `classNames + twMerge` then just use `twMerge` as it can do both I think?